application/features/mfa.feature

```diff
@@ -61,7 +61,6 @@ Feature: MFA
     Then the response status code should be 400
 
   Scenario: Create new MFA record of type webauthn
-#TODO - create a test double for the webauthn client
```

Comment:
Does this TODO still need to be here, perhaps with a different comment, since the scenario isn't implemented?